examples/switch/setup.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+import io
+import re
+from setuptools import setup, find_packages
+import sys
+
+with io.open('./switch/__init__.py', encoding='utf8') as version_file:
+    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]", version_file.read(), re.M)
+    if version_match:
+        version = version_match.group(1)
+    else:
+        raise RuntimeError("Unable to find version string.")
+
+
+with io.open('README.rst', encoding='utf8') as readme:
+    long_description = readme.read()
+
+
+setup(
+    name='switch',
+    version=version,
+    description='A demonstration of all features of a toga.Switch example for the native GUI toolkit, Toga.',
+    long_description=long_description,
+    author='DariusMontez',
+    author_email='darius.montez@gmail.com',
```

Comment:
I think we're preferring 'BeeWare' and 'contact@pybee.org' for these entries, but this is more of a nit than a blocker.

Reply:
Not a problem. Thank's for letting me know.